docs/testrunner-toolkit/configuration/cypress.md

```diff
@@ -75,6 +76,16 @@ Instructs `saucectl` run tests remotely through Sauce Labs (`sauce`) or locally
 ```
 ---
 
+### `timeout`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Instructs how long `saucectl` should wait for the suites to complete. You can override this setting for individual suites using the `timeout` setting within the [`suites`](#suites) object. If not set, the default value is `0` (unlimited).
```

Comment:
I know that technically the format is "string", but in the CLI doc, we show the format as `DURATION`, which might more accurately reflect that the value is expected to be some integer + a unit of time. What do you think? Also, we should state what units of time are supported -- is it only minutes? If the value does not include the unit (e.g., is just `15`), is it still valid?

Reply:
This is a value can be defined with different units (I need to update the message).
It's that format: `10ms`, `10s`, `10m`, `10h`.
It allows a wide range of configuration.

Feel free to suggest any wording that you find best !

Reply:
```suggestion
<p><small>| OPTIONAL | DURATION |</small></p>

Instructs how long (in `ms`, `s`, `m`, or `h`) `saucectl` should wait for the suites to complete. You can override this setting for individual suites using the `timeout` setting within the [`suites`](#suites) object. If not set, the default value is `0` (unlimited).
```

Reply:
I would also like clarification on:
1. What happens if someone puts only a number with no unit? Does it default to "m" or is it invalid?
2. Is the defaults.timeout setting the time limit for ALL suites to complete or for each suite? So if I say `15m`, does each suite get 15m, or do they all have to be done within 15m? This matter in terms of the use of the work "override" in the description, so I will provide an updated suggestion based on your answer.

Reply:
@sweeneyskirt-sl 
1. No unit means `ns`. We may want to make it seconds by default, but this is not the case currently).
2. This is a default value, that mean it a particular suite does not have a `timeout` value, it will be using the default one. Hence, it's only a per-suite timeout. 15min timeout + 2 suites + 1 concurrency = max 30min execution.